pkg/node/sendoperation/callsc/callsc.go

```diff
@@ -47,11 +54,16 @@ func New(address string, function string, parameters []byte, gasLimit uint64, co
 }
 
 func (c *CallSC) Content() interface{} {
+	addressString, err := utils.DeserializeAddress(c.address)
+	if err != nil {
+		return nil
```

Comment:
actually we don't need it, since we deserialize the same address that was successfully serialized in creation of the operation in. 
`func New(address string, function string, parameters []byte, gasLimit uint64, coins uint64,
) (*CallSC, error) {
	versionedAddress, err := utils.SerializeAddress(address)` so no need to check this error , it's an unreachable code.

Reply:
no, you can create an operation in another software that Massa Station (massa-web3 for example). We should not do the hypothesis of unreachable code. When you call `c.Content()` we don't want to do this:

```go
content := c.Content()
if content == nil {
  panic("...")
}
```

but rather:

```go
content, err := c.Content()
if err != nil {
  panic("...")
}
```